mlflow/recipes/utils/metrics.py

```diff
@@ -37,7 +37,7 @@ def from_custom_metric_dict(cls, custom_metric_dict):
         )
 
 
-BUILTIN_CLASSIFICATION_RECIPE_METRICS = [
+BUILTIN_BINARY_CLASSIFICATION_RECIPE_METRICS = [
     RecipeMetric(name="true_negatives", greater_is_better=True),
```

Comment:
Those metrics need to know which class is positive, which should be specified by the user.

Reply:
I have removed them from the multi-class dictionary.